compute-aggregated.js

```diff
@@ -24,11 +32,16 @@ elmApp.ports.export.subscribe(
     foodProcessesOnlyAggregated,
   }) => {
     try {
-      exportJson("public/data/textile/processes_impacts.json", textileProcesses);
-      exportJson("public/data/food/processes_impacts.json", foodProcesses);
+      exportJson(process.env.TEXTILE_PROCESSES_IMPACTS_PATH, textileProcesses);
+      exportJson(process.env.FOOD_PROCESSES_IMPACTS_PATH, foodProcesses);
       exportJson("public/data/textile/processes.json", textileProcessesOnlyAggregated);
       exportJson("public/data/food/processes.json", foodProcessesOnlyAggregated);
-      console.log("EXPORTED!");
+      console.log(
+        `\n4 files exported to\n- ${process.env.TEXTILE_PROCESSES_IMPACTS_PATH}\n- ${process.env.FOOD_PROCESSES_IMPACTS_PATH}\n- public/data/textile/processes.json\n- public/data/food/processes.json!`,
+      );
+      console.log(
+        "\n🚨 Be sure to commit the detailed impacts `processes_impacts.json` files in the `ecobalyse-private` repo.",
+      );
```

Comment:
Same remark as per using a single template strings with natural newlines for legibility and use a single `console.log` call.